### DISCUSSION TITLE
[v5] how use "useConnection" in Many to Many relationship

### DISCUSSION THREAD

**Discussion by jairochaves2** [adonisjs/core#3001] [Help]
How to use `post.useConnection('user_db_connection').load('users')` in Many to Many relationship with pivot table in local database?

I tried to use it, but it looks for the pivot table in the user database, on another server, meanwhile the pivot table is in the local database.

**Comment by McSneaky** [comment]
> How to use `post.useConnection('user_db_connection').load('users')` in Many to Many relationship with pivot table in local database?
> 
> I tried to use it, but it looks for the pivot table in the user database, on another server, meanwhile the pivot table is in the local database.

AFAIK most databases doesn't support cross-server joins. So it's not supported
